ush/ufsda/stage.py

```diff
@@ -33,21 +33,31 @@ def gdas_fix(input_fix_dir, working_dir, config):
     # create output directories
     ufsda.disk_utils.mkdir(config['fv3jedi_fieldmetadata_dir'])
     ufsda.disk_utils.mkdir(config['fv3jedi_fix_dir'])
-    # figure out analysis resolution
-    if config['DOHYBVAR']:
-        case_anl = config['CASE_ENKF']
-    else:
-        case_anl = config['CASE_ANL']
+
+    # error checking
+    dohybvar = config['DOHYBVAR']
+    case = config['CASE']
+    case_enkf = config['CASE_ENKF']
+    case_anl = config['CASE_ANL']
+    if dohybvar and not case_enkf == case_anl:
+        raise ValueError(f"dohybvar is '{dohybvar}' but case_enkf= '{case_enkf}' does not equal case_anl= '{case_anl}'")
+
+    # set layers
     layers = int(config['LEVS'])-1
 
     # figure out staticb source
     staticb_source = config.get('STATICB_TYPE', 'gsibec')
 
-    # link staticb
-    if staticb_source in ['bump', 'gsibec']:
+    # link bump staticb
+    if staticb_source in ['bump']:
```

Comment:
@RussTreadon-NOAA I think I remember seeing some discussion about the gsibec fix file directory structure? 

what about going back to `if staticb_source in ['bump', 'gsibec']:` but making sure `case_anl` is consistent regardless of BUMP/GSIBEC? I would think we should be able to have independent `case` and `case_anl` if possible.

Reply:
Hmm, I'll need to think about this.   The current approach gets back to closed issue #403.

The way gsibec is implemented in UFS-DA we need the gsibec `case` to equal the background `case`.   The `case` gsibec may be used to generate a variational analysis on the `case_anl` grid.    The gsibec logic in `stage.py` is designed for this flexibility.   gsibec is associated with `case` while the analysis resolution is specified by `case_anl`.

Reply:
yeah I think this is exactly what I had in mind, this approach reduces the amount of redundant code in the "if-block"

Reply:
Got it.  The suggested change was made and committed at 581bdd8.